glib/src/types.rs

```diff
@@ -246,6 +246,18 @@ impl StaticType for Type {
     }
 }
 
+pub trait StaticTypeExt {
+    /// Ensures that the type has been registered with the type system.
+    fn type_ensure();
+}
+
+impl<T: StaticType> StaticTypeExt for T {
+    #[doc(alias = "g_type_ensure")]
```

Comment:
This needs to be on the trait definition, not on the implementation

Reply:
Done